conan/cli/commands/create.py

```diff
@@ -62,6 +66,21 @@ def create(conan_api, parser, *args):
     lockfile = conan_api.lockfile.update_lockfile_export(lockfile, conanfile, ref, is_build)
 
     print_profiles(profile_host, profile_build)
+    if profile_host.runner and not os.environ.get("CONAN_RUNNER_ENVIRONMENT"):
+        try:
+            runner_type = profile_host.runner['type']
```

Comment:
```suggestion
            runner_type = profile_host.runner['type'].lower()
```

So that maybe users that try with `Docker` (ie, me!) work?